tests/memory_usage_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+TEMP_DIR=$(mktemp -d)/aktualizr
+MEMMORY_LIMIT=104857600 #100 Mib
```

Comment:
It looks like the memory usage continually fails with 457 Mib. What happens if we try running this with a limit of 500 Mib?

Reply:
It will pass